internal/controller/api/v1beta1/infraenv_types.go

```diff
@@ -110,6 +110,7 @@ type InfraEnvStatus struct {
 	// ISODownloadURL specifies an HTTP/S URL that contains a discovery ISO containing the
 	// configuration from this InfraEnv.
 	ISODownloadURL string                   `json:"isoDownloadURL,omitempty"`
+	CreatedAt      *metav1.Time             `json:"createdAt,omitempty"`
```

Comment:
suggest `CreatedTime` based on the [api convention](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions): "The name of a field that specifies the time at which something occurs should be called `somethingTime`."

Reply:
nevermind, I'll take this to that PR which I see now that you are waiting on.

Reply:
Correct, I just have this commit so that my PR even compiles, but it’ll disappear once I rebase